llvm/lib/SYCLLowerIR/CompileTimePropertiesPass.cpp

```diff
@@ -332,6 +332,16 @@ parseSYCLPropertiesString(Module &M, IntrinsicInst *IntrInst) {
   return result;
 }
 
+template <typename T>
+void searchUserIgnoringCast(Value *V, SmallVector<Instruction *, 4> &List) {
```

Comment:
nit: please consider renaming. May be getUserListIgnoringCast? Also, please add a short comment about it.

Thanks

Reply:
done